src/Application_controller.C

```diff
@@ -223,41 +216,36 @@ void Application_controller::handle_internal_path(
             setInternalPath(url.str(), true);
         };
 
-        if (false) { }
+        if (false) {
+        }
 
-            // /hw
+        // /hw
         else if (internal_path == Path::hw) {
             transaction.commit();
             switch (current_user->role()) {
-                case User::Role::Student:
-                    add_tilde_user();
-                    break;
-
-                case User::Role::Grader:
-                    permission_denied("There's nothing for you here.");
-                    break;
-
-                case User::Role::Admin:
-                    set_page_title("Edit Assignments");
-                    set_new_widget<Assignments_view>(session_);
-                    break;
+            case User::Role::Student: add_tilde_user(); break;
+
+            case User::Role::Grader:
+                permission_denied("There's nothing for you here.");
+                break;
+
+            case User::Role::Admin:
+                set_page_title("Edit Assignments");
+                set_new_widget<Assignments_view>(session_);
+                break;
             }
 
             // /
         } else if (internal_path == Path::root) {
             transaction.commit();
             switch (session_.user()->role()) {
-                case User::Role::Student:
-                    setInternalPath(current_user->hw_url(), true);
-                    break;
+            case User::Role::Student:
+                setInternalPath(current_user->hw_url(), true);
+                break;
 
-                case User::Role::Grader:
-                    setInternalPath(Path::grade, true);
-                    break;
+            case User::Role::Grader: setInternalPath(Path::grade, true); break;
```

Comment:
Some of this configuration I gave you is not very good. Ew. I guess I can live with this?